src/gpuocean/utils/NetCDFInitialization.py

```diff
@@ -132,6 +153,76 @@ def getBoundaryConditionsData(source_url_list, timestep_indices, timesteps, x0,
     return bc_data
 
 
+def getWindSourceterm(source_url_list, timestep_indices, timesteps, x0, x1, y0, y1):
+    """
+    timestep_indices => index into netcdf-array, e.g. [1, 3, 5]
+    timestep => time at timestep, e.g. [1800, 3600, 7200]
+    """
+    
+    if type(source_url_list) is not list:
+        source_url_list = [source_url_list]
+    
+    num_files = len(source_url_list)
+    
+    source_url = source_url_list[0]
+    
+    assert(num_files == len(timesteps)), str(num_files) +' vs '+ str(len(timesteps))
+    
+    if (timestep_indices is None):
+        timestep_indices = [None]*num_files
+        for i in range(num_files):
+            timestep_indices[i] = range(len(timesteps[i]))
+        
+    u_wind_list = [None]*num_files
+    v_wind_list = [None]*num_files
+    
+    if "Uwind" in Dataset(source_url_list[0]).variables:
+        for i in range(num_files):
+            try:
+                ncfile = Dataset(source_url_list[i])
+                if i == 0 and len(ncfile.variables['Uwind'].shape) == 0:
+                    return WindStress.WindStress()
+                u_wind_list[i] = ncfile.variables['Uwind'][timestep_indices[i], y0:y1, x0:x1]
+                v_wind_list[i] = ncfile.variables['Vwind'][timestep_indices[i], y0:y1, x0:x1]
+            except Exception as e:
+                raise e
+            finally:
+                ncfile.close()
+    else:
+        return WindStress.WindStress()
+
+    u_wind = u_wind_list[0].filled(0)
+    v_wind = v_wind_list[0].filled(0)
+    for i in range(1, num_files):
+        u_wind = np.concatenate((u_wind, u_wind_list[i].filled(0)))
+        v_wind = np.concatenate((v_wind, v_wind_list[i].filled(0)))
+    
+    u_wind = u_wind.astype(np.float32)
+    v_wind = v_wind.astype(np.float32)
+    
+    wind_speed = np.sqrt(np.power(u_wind, 2) + np.power(v_wind, 2))
+
+    # C_drag as defined by Engedahl (1995)
+    #(See "Documentation of simple ocean models for use in ensemble predictions. Part II: Benchmark cases"
+    #at https://www.met.no/publikasjoner/met-report/met-report-2012 for details.) /
+    def computeDrag(wind_speed):
+        C_drag = np.where(wind_speed < 11, 0.0012, 0.00049 + 0.000065*wind_speed)
+        return C_drag
+    C_drag = computeDrag(wind_speed)
+
+    rho_a = 1.225 # Density of air
+    rho_w = 1025 # Density of water
+
+    #Wind stress is then 
+    # tau_s = rho_a * C_drag * |W|W
+    wind_stress = C_drag * wind_speed * rho_a / rho_w
+    wind_stress_u = wind_stress*u_wind
+    wind_stress_v = wind_stress*v_wind
+    
```

Comment:
Is there a reason for computing windstress here, rather than using the function in the WindStress class?

Reply:
This entire function consists of duplicates and is actually never used: The function `getWind` does the same job (without repeating the stress calculation) and is also actually used.